diff --git a/lib/transformer.js b/lib/transformer.js
index 798a996..31155fd 100644
--- a/lib/transformer.js
+++ b/lib/transformer.js
@@ -822,7 +822,7 @@ class Transformer {
                 enumName);
                 await (0, writeFileSafely_1.writeFileSafely)(path_1.default.join(Transformer.getSchemasPath(), `enums/${fileName}`), `${this.generateImportZodStatement()}\n` +
                     `export const ${schemaExportName} = z.enum([${values.map((v) => `'${v}'`).join(', ')}])\n\n` +
-                    `export type ${enumName} = z.infer<typeof ${schemaExportName}>;`);
+                    `export type ${enumName}Enum = z.infer<typeof ${schemaExportName}>;`);
             }
         }
     }
